share/messages_to_error_log.txt

```diff
@@ -11771,6 +11771,9 @@ ER_SEC_LOG_PATH_NULL
 ER_KEYRING_COMPONENT_KMS_ERROR
   eng "KMS communication error: %s"
 
+ER_XB_MSG_DOWNGRADING_LOG_FILE
+  eng "Upgrade failed, cleaning up redo logs, so server can start with previous version"
```

Comment:
Upgrade failed, recreating redo logs with the previous version

Reply:
fixed